app/assets/stylesheets/content/work_packages/resizer/resizer.sass

```diff
@@ -1,6 +1,7 @@
 .work-packages--resizer
```

Comment:
Would it be possible to make the handle be "grabable" everywhere on the vertical lign? And showing the handle icon only in the middle? This could improve discoverability as the mouse cursor changes on hover.

Reply:
Yeah, we thought about that, too. However in my eyes it would look strange if the cursor changes somehow out of nowhere. Especially on high screens this might confuse..

Reply:
Do you thinks so? Isn't that the default behaviour in UI, that borders are draggable? In my Mac Mail it is definitely like that.

Reply:
Yes, if you make the change directly on the border I agree. In this case there is perhaps even not really a need for an icon any more. 
But I think that the implementation of this is rather ugly. We would need to increase the component to the whole height and the force its position over the border. Or Is there a cleaner way I am missing?